### ISSUE TITLE
Bug: Conditional Logic Plugin (1.5.6) causing JS error

### ISSUE THREAD

**Issue by wgstjf** [wpmetabox/meta-box#1250]
Hi guys,

We have a site with latest version of MB Conditional Logic install (Version 1.5.6) and it's causing JS errors in the admin area.

Clicking anywhere within the viewport of a page generates an 'unrecognized expression' error.  If I disable the plugin or roll it back to an older version the JS errors don't appear.

A couple of typical example errors are as follows:

```
Uncaught Error: Syntax error, unrecognized expression: ,.add-clone
load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,utils,moxiejs,plupload&ver=4.9.7:formatted:565 
    at Function.fa.error (load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,utils,moxiejs,plupload&ver=4.9.7:formatted:565)
    at fa.tokenize (load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,utils,moxiejs,plupload&ver=4.9.7:formatted:893)
    at fa.compile (load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,utils,moxiejs,plupload&ver=4.9.7:formatted:1051)
    at fa.select (load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,utils,moxiejs,plupload&ver=4.9.7:formatted:1089)
    at Function.fa (load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,utils,moxiejs,plupload&ver=4.9.7:formatted:334)
    at Function.a.find (load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,utils,moxiejs,plupload&ver=4.9.7:formatted:4637)
    at HTMLDocument.handlers (load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,utils,moxiejs,plupload&ver=4.9.7:formatted:2134)
    at HTMLDocument.dispatch (load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,utils,moxiejs,plupload&ver=4.9.7:formatted:2109)
    at HTMLDocument.r.handle (load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,utils,moxiejs,plupload&ver=4.9.7:formatted:1996)
fa.error @ load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,utils,moxiejs,plupload&ver=4.9.7:formatted:565
fa.tokenize @ load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,utils,moxiejs,plupload&ver=4.9.7:formatted:893
fa.compile @ load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,utils,moxiejs,plupload&ver=4.9.7:formatted:1051
fa.select @ load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,utils,moxiejs,plupload&ver=4.9.7:formatted:1089
fa @ load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,utils,moxiejs,plupload&ver=4.9.7:formatted:334
a.find @ load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,utils,moxiejs,plupload&ver=4.9.7:formatted:4637
handlers @ load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,utils,moxiejs,plupload&ver=4.9.7:formatted:2134
dispatch @ load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,utils,moxiejs,plupload&ver=4.9.7:formatted:2109
r.handle @ load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,utils,moxiejs,plupload&ver=4.9.7:formatted:1996
```

```
Error: Syntax error, unrecognized expression: ,[name="post_type"],[name="page_primary_button1_type"],[name="page_primary_button2_type"],[name*="page_type"],[name*="textimage_media"],.add-clone
load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,utils,moxiejs,plupload&ver=4.9.7:2 Uncaught 
    at Function.fa.error (load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,utils,moxiejs,plupload&ver=4.9.7:2)
    at fa.tokenize (load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,utils,moxiejs,plupload&ver=4.9.7:2)
    at fa.compile (load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,utils,moxiejs,plupload&ver=4.9.7:2)
    at fa.select (load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,utils,moxiejs,plupload&ver=4.9.7:2)
    at Function.fa (load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,utils,moxiejs,plupload&ver=4.9.7:2)
    at Function.a.find (load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,utils,moxiejs,plupload&ver=4.9.7:9)
    at HTMLDocument.handlers (load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,utils,moxiejs,plupload&ver=4.9.7:3)
    at HTMLDocument.dispatch (load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,utils,moxiejs,plupload&ver=4.9.7:3)
    at HTMLDocument.r.handle (load-scripts.php?c=0&load[]=jquery-core,jquery-migrate,utils,moxiejs,plupload&ver=4.9.7:3)
```

Love to get this fixed asap as this is affecting live sites.

Cheers,

Will

**Comment by rilwis** [comment]
Hi Will, I can't replicate the bug. Can you send me the code to register meta box?

**Comment by rilwis** [comment]
Hi Will,

I just found the bug and fixed it in [version 1.5.7](https://metabox.io/plugins/meta-box-conditional-logic/). Please update.

**Comment by wgstjf** [comment]
Sorry for the delay in coming back to you, I've been away from my desk.  Great news you've got it fixed, thanks as always.

Will
